test/e2e/antreapolicy_test.go

```diff
@@ -1039,30 +1093,65 @@ func testACNPClusterGroupRefRulePodAdd(t *testing.T, data *TestData) {
 	executeTestsWithData(t, testCase, data)
 }
 
+// There are three situations of a Pod's IP(s):
+// 1. Only one IPv4 address.
+// 2. Only one IPv6 address.
+// 3. One IPv4 and one IPv6 address, and we don't know the order in list.
+// We need to add all IP(s) of Pods as CIDR to IPBlock.
```

Comment:
the comment doesn't match what the function is doing

Reply:
Fixed